docs/install.md

```diff
@@ -1,27 +1,32 @@
 # Installation/Update
 
 ## Dependencies
-* PHP >= 7.0
-* Nextcloud 14
-* libxml >= 2.7.8 (2.9 recommended)
-* php-curl
+* PHP >= 7.2
+* Nextcloud 20
+* libxml >= 2.7.8
+You also need some PHP extensions:
+* json
+* simplexml
+* xml
+* dom
+* curl
 * iconv
-* SimpleXML
 
 ## Supported Databases
-* PostgreSQL >= 9.4 (recommended)
-* MariaDb >= 5.5
+* PostgreSQL >= 10
+* MariaDB >= 10.2
+* MySQL >= 8.0
 * SQLite (discouraged)
 
 ## Performance Notices
```

Comment:
Can we link this to the upstream documentation?

Reply:
Yea good idea :)